app/src/main/java/com/example/androidthings/assistant/LedBlinkThread.java

```diff
@@ -0,0 +1,68 @@
+package com.example.androidthings.assistant;
+
+import android.util.Log;
+
+import com.google.android.things.pio.Gpio;
+
+import java.io.IOException;
+import java.util.Random;
+
+public class LedBlinkThread extends Thread {
+
+    private static final String TAG = "LedBlinkingThread";
```

Comment:
Typically we use `LedBlinkThread.class.getSimpleName()`

Reply:
good point. I changed it.